scripts/linters/run_lint_checks_test.py

```
@@ -25,7 +25,7 @@
 
 from core.tests import test_utils
 
-from typing import List, Optional
+from typing import Dict, List, Optional
```

Ruff (0.15.11)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)